src/ControlSystem/SettleToConstant.hpp

```diff
@@ -9,18 +9,21 @@
 #include "ControlSystem/FunctionOfTime.hpp"
 #include "DataStructures/DataVector.hpp"
 
-/// \ingroup ControlSystemGroup
-/// Contains functions of time to support the dual frame system.
 namespace FunctionsOfTime {
 /// \ingroup ControlSystemGroup
-/// Sets f(t) and derivatives via exponentials, such that f(t)
-/// approaches a constant in time.
-/// Function is f(t) = A + (B+C(t-t0)) exp(-(t-t0)/tau)
-/// where A,B,C,t0,tau are constants.
+/// Given an initial function f(t) and its first two derivatives
```

Comment:
replace `match_time` with t_0 and `decay_time` with \tau (except for the last line) and then latex f(), g(), A, B, C, and t>t_0